docs/compass/03-packages-api.md

```diff
@@ -0,0 +1,81 @@
+# Packages API
+
+## Introduction
+
+On Kyma Runtime, every Application is represented as a single Service Class, and every Package of a given Application is represented as a single Service Plan.
+
+![API Packages Diagram](./assets/packages-api.svg)
+
+This document describes API for Packages, which groups multiple API Definitions, Event Definitions and Documents.
+
+## Assumptions
+
+- A single API or Event Definition can be a part of a single Package.
+- Package belongs to a single Application entity.
```

Comment:
where can I find information that Package is responsible for storing Auth for all API that belongs to the package?
Or, in other words, that API in package share the same credentials?

Reply:
In assumptions for credential requests